### PR TITLE
Fixed modal logic so it works with features

### PR DESCRIPTION
**Pull request by kate-darby**
This took.... less time than I thought it would. I reworked the logic a bit and it is working with features. @AlexArms @chrimack @sunkim0330

### CODE REVIEW

**Review by sunkim0330 — APPROVED** [review]
I just checked and everything looks good! but I think my merge messed up your pull request. I'm so sorry  😟 I shouldve wait until you merge yours

**Comment by kate-darby** [comment]
It's fine! It was one line. Super Easy to fix.

**Pull request merged**
